Packs/PAN-OS/ReleaseNotes/1_8_0.md

```diff
@@ -0,0 +1,5 @@
+#### Integrations
+##### Palo Alto Networks PAN-OS
+- Updated the Docker image to: *demisto/python3:3.9.7.24076*.
+- Added the new argument **polling** to the commands: ***panorama-commit***, ***panorama-download-latest-content-update***, ***panorama-install-latest-content-update***, ***panorama-download-panos-version***, ***panorama-install-panos-version***, and ***panorama-install-file-content-update***. 
+When **polling** is used, the command will try to return the results. The new commands allow users to run polling with a single command, that doesn't require ***GenericPolling Playbook***.
```

Comment:
```suggestion
When **polling** is used, the command will try to return the results. The commands allow users to run polling with a single command, that doesn't require ***GenericPolling Playbook***.
```

Generally trying to stay away from time relative language (now, new, later, upcoming, etc.)